rep-2005.rst

```diff
@@ -287,6 +287,7 @@ All items on the list for the next distro should already be released into the ro
   * `rclcpp/rclcpp_components <https://index.ros.org/p/rclcpp_components/>`_
   * `rclcpp/rclcpp_lifecycle <https://index.ros.org/p/rclcpp_lifecycle/>`_
   * `rclpy/rclpy <https://index.ros.org/p/rclpy/>`_
+  * `rclc/rclc <https://index.ros.org/p/rclc/>`_
```

Comment:
rclc should be a different PR in my opinion, this isn't directly related to micro-ros alone.

Reply:
I'll create a separate PR, but please note that several micro-ROS packages in this PR depend on rclc.